firebase/functions/src/comments.ts

```diff
@@ -85,4 +108,20 @@ function addToFIreBase(data: any, collection: any) {
       });
   });
 }
-//943.05
+
+function update(data: any, collection: any,id:any) {
```

Comment:
`CollectionReference` and `DocumentReference` only accept `string` as parameters.

We could be strict with the data received but it is not necessary. In case that we need to, `data` must use the data type `UpdateData` which is described like so:

```typescript
type UpdateData = {[fieldPath: string]: any};
```

Suggested changes (ignoring type for `data` argument) are:
```suggestion
function update(data: any, collection: string, id: string) {
```